topbar.js

```diff
@@ -539,13 +539,14 @@ function enable () {
 
     signals.connect(Settings.settings, 'changed::topbar-follow-focus', (settings, key) => {
         let monitors = Tiling.spaces.monitors;
-        let to = setMonitor(Main.layoutManager.focusMonitor);
-        let space = monitors.get(to);
-        updateWorkspaceIndicator(space.workspace.index());
```

Comment:
We still want these lines too. `setMonitor` updates `panelMonitor` which the tiling code uses to know which monitor the panel is located. `updateWorkspaceIndicator` refreshes the workspace name in the top bar.